src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
Ok, do you have any reference on how to point to a package repository on Appveyor? Was looking for that now, but could not find any. BTW, as a matter of fact, the current [last commit](https://github.com/brendankowitz/Caliburn.Micro.Autofac/pull/5/commits/12c79d40ac1e6e23cf6a5bdcc34992e7e2951efb) now fails on CI because it cannot find that locally built package anymore.